core-java-modules/core-java-io/src/test/java/com/baeldung/extension/ExtensionFromMimeTypeUnitTest.java

```diff
@@ -0,0 +1,64 @@
+package com.baeldung.extension;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import org.apache.tika.mime.MimeTypeException;
+
+import org.junit.Test;
+
+import com.j256.simplemagic.ContentInfo;
+
+public class ExtensionFromMimeTypeUnitTest {
+    private static final String IMAGE_JPEG_MIME_TYPE = "image/jpeg";
+    @Test
+    public void whenUsingTika_thenGetFileExtension() throws MimeTypeException {
+        List<String> expectedExtensions = Arrays.asList(".jpg", ".jpeg", ".jpe", ".jif", ".jfif", ".jfi");
+        org.apache.tika.mime.MimeTypes allTypes = org.apache.tika.mime.MimeTypes.getDefaultMimeTypes();
+        org.apache.tika.mime.MimeType type = allTypes.forName(IMAGE_JPEG_MIME_TYPE);
+        String primaryExtension = type.getExtension();
+        assertEquals(".jpg", primaryExtension);
+        List<String> detectedExtensions = type.getExtensions();
+        assertThat(detectedExtensions).containsExactlyElementsOf(expectedExtensions);
+    }
+
+    @Test
+    public void  whenUsingJodd_thenGetFileExtension() {
+        String[] expectedExtensions = {"jpeg","jpg","jpe"};
+        String[] detectedExtensions = jodd.net.MimeTypes.findExtensionsByMimeTypes(IMAGE_JPEG_MIME_TYPE, false);
+        assertThat(detectedExtensions).containsExactly(expectedExtensions);
+    }
+
+    @Test
+    public void whenUsingMimetypesFileTypeMap_thenGetFileExtension() {
+        String[] expectedExtensions = {"jpeg","jpg","jpe"};
```

Comment:
Since using AssertJ now, how about we use the same List<String> approach in each test? This will result in less differences for the reader to understand between the examples
```suggestion
        List<String> expectedExtensions = Arrays.asList(".jpeg", ".jpg", ".jpe");
```

Reply:
Done